HedgeLib/src/archives/hl_pacx.c

```diff
@@ -1866,14 +1962,24 @@ static HlResult hlINPACxV4DecompressToBlobs(HlBlob* HL_RESTRICT pac,
         /* Setup split PAC entries, decompressing if necessary. */
         for (i = 0; i < splitTable->splitCount; ++i)
         {
-            const HlPACxV4Chunk* chunks = (const HlPACxV4Chunk*)
-                hlOff64Get(&splitEntries[i].chunksOffset);
+            if (isV403)
+            {
```

Comment:
Since this starts a new scope this would be a good place to cast the pointer like I mentioned above

```c
HlPACxV403SplitEntry* splitEntriesV403 = (HlPACxV403SplitEntry*)splitEntries;
```